nondex-gradle-plugin/src/main/groovy/edu/illinois/nondex/gradle/plugin/tasks/NonDexTest.groovy

```diff
@@ -11,7 +11,7 @@ class NonDexTest extends Test {
 
     void init(Configuration config) {
         project.extensions.create("nondexTest", NonDexTestExtension)
-        project.extensions.nondexTest.outPath = project.buildDir.absolutePath + "/out.jar"
+        project.extensions.nondexTest.outPath = project.buildDir.absolutePath + File.pathSeparator + "out.jar"
```

Comment:
I've made that change in my last commit. Are you looking at the newest version of this file?